install-dev/install_version.php

```diff
@@ -24,7 +24,7 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
-define('_PS_INSTALL_VERSION_', '8.1.0');
+define('_PS_INSTALL_VERSION_', '9.0.0');
```

Comment:
Can we use the constant version inside the `src/Core/Version.php` class ?

Reply:
If we can centralize this information it would be better. I agree

Reply:
I tried but it doesn't work https://github.com/PrestaShop/PrestaShop/actions/runs/5124359330/jobs/9216102447?pr=31146

I added a comment so we don't try this on every update 😅